README.md

```diff
@@ -43,9 +43,11 @@ first in order to allow VS Code to detect the language.
 Language | Dash Docset Keys | Docset Setting | Language Plugin |
 ------------ | ------------- | ------------- | :-------------: |
 Ansible | ansible | dash.languageIdToDocsetMap.ansible, dash.languageIdToDocsetMap.ansible-advanced | [link](https://marketplace.visualstudio.com/items?itemName=haaaad.ansible)
+Arduino | arduino | dash.languageIdToDocsetMap.arduino | [link](https://marketplace.visualstudio.com/items?itemName=vsciot-vscode.vscode-arduino)
```

Comment:
we shouldn't add Arduino here because this is for `language to docset mapping`. Since, we use `filename to docset mapping`, we should put `Arduino` in https://github.com/deerawan/vscode-dash#file-name-to-docset-matching

Reply:
I changed it :)